bin/package_build.py

```diff
@@ -273,66 +273,35 @@ def getIBMValidatedOpenSourceList(oskey):
 		print("Couldn't pull. Error: ",str(e))
 	else:
 		data_json = json.loads(data)
+		osversions = ['SLES 12.x', 'SLES 15.x', 'Ubuntu 20.x', 'Ubuntu 22.x', 'RHEL 8.x/7.x', 'RHEL 9.x']
+        for oskey in os_versions:
```

Comment:
Please switch to using tabs for the indentation, it's throwing errors for me when I run it.